Examples/MAX32665/Demo/README.md

```diff
@@ -0,0 +1,41 @@
+## Description
+
+A basic Hello_World program that use [LVGL Graphics Library](https://lvgl.io/).
+
+This example
+-   Works in EvKit-V1 Board that has [LS013B7DH03 Monochrome LCD Display](https://www.sharpsde.com/products/displays/model/ls013b7dh03/).
+-   Uses LVGL graphic library to generate ADI Logo and prints the count.
+
+To create LVGL compatible fonts and images use [Font Converter](https://lvgl.io/tools/fontconverter) and [Bitmap Converter](https://lvgl.io/tools/imageconverter)
+To create LVGL compatible screens with drag & drop use [SquareLine GUI Builder](https://squareline.io/)
+
+Please check project.mk to learn how compile lvgl in MSDK
+
+
+## Software
+
+### Project Usage
+
+Universal instructions on building, flashing, and debugging this project can be found in the **[MSDK User Guide](https://analog-devices-msdk.github.io/msdk/USERGUIDE/)**.
+
+### Project-Specific Build Notes
+
+(None - this project builds as a standard example)
```

Comment:
"this project is only supported on the [MAX32665EVKIT](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/max32665evkit.html)"